x-pack/plugins/security_solution_serverless/server/endpoint/services/metering_service.ts

```diff
@@ -116,12 +121,18 @@ export class EndpointMeteringService {
       },
       source: {
         id: taskId,
-        instance_group_id: projectId,
+        instance_group_id: projectId || METERING_TASK.MISSING_PROJECT_ID,
         metadata: {
           tier: this.tier,
         },
       },
     };
+
+    if (!projectId) {
+      logger.error(`project id missing for record: ${usageRecord}`);
```

Comment:
what is output to the log for `usageRecord` here (is it `[object object]`)? Wondering if you should `JSON.stringify(usageRecord)`

Reply:
well that's embarrassing. maybe I should have stayed on vacation 🤣. fixed, thanks for spotting.

Reply:
😆 

yeah, I wish I was still on vacation too.